frontend/public/index.html

```diff
@@ -2,48 +2,162 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <title>Penn Course Review</title>
+
+    <meta
+      name="description"
+      content="Penn Course Review provides quantitative data on courses and professors to help students make informed decisions about coursework. It includes metrics for course difficulty and quality, professor quality, and more."
+    />
+    <meta
+      name="keywords"
+      content="penn, course, review, pcr, upenn, university, pennsylvania, class, classes, best, worst, hardest"
+    />
+    <meta name="author" content="Penn Labs <contact@pennlabs.org>" />
+
+    <meta property="og:title" content="Penn Course Review" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://penncfa.com" />
```

Comment:
This probably shouldn't be `https://penncfa.com`? @ccabo1

Reply:
....yep; changing rn

Reply:
@ezwang fixed, good flag 😓